magicctapipe/scripts/lst1_magic/magic_calib_to_dl1.py

```diff
@@ -145,8 +146,6 @@ def magic_calib_to_dl1(input_file, output_dir, config, process_run=False):
             logger.warning('\nHot pixels do not exist in a simulation. Setting the "find_hotpixels" option to False...')
             config_cleaning.update({'find_hotpixels': False})
 
-        unsuitable_mask = None
```

Comment:
is this change intended?
it looks not related to the dead time

Reply:
Yes indeed it is not related. I recovered back to the original one.